src/cfg.c

```diff
@@ -34,7 +34,7 @@
 #include "tt_static.h"
 #include "lua/uri.h"
 
-enum { MAX_OPT_NAME_LEN = 256, MAX_OPT_VAL_LEN = 256 };
+enum { MAX_OPT_NAME_LEN = 256, MAX_OPT_VAL_LEN = 512 };
```

Comment:
Is the max length documented? If not, then the doc bot request isn't required. OTOH, a changelog entry would be nice to have anyway.

Reply:
Thank you for the review! I couldn't find anything about max parameter length. Removed doc-request and added a changelog.